frontend/src/components/organisms/MatchesList/MatchesList.tsx

```diff
@@ -93,7 +141,43 @@ const returnMatchListItemsFromMatchesListDto = (matchesListDto: MatchesListDto):
             currentPlayerLevel: undefined,
             currentMatchNumber: undefined,
          }
-
-      } as MatchListItem
+      }
    })
-}
\ No newline at end of file
+}
+
+
+const getMatchInformationDto = (matchId: string): Promise<MatchInformationDto> => {
+   return MatchInformationRestApi()
+      .getMatchesList(matchId)
+      .then(matchInformationDto => {
+         return {
+            matchId: matchInformationDto.matchId,
+            firstMatchSideId: matchInformationDto.firstMatchSideId,
+            secondMatchSideId: matchInformationDto.secondMatchSideId,
+            winnerId: matchInformationDto.winnerId
+         }
+      });
+}
+
+const getPlayerProfileDto = (playerId: string): Promise<PlayerProfileDto> => {
+   return UserProfileRestApi()
+      .getPlayerProfile(playerId)
+      .then(playerProfileDto => {
+         return {
+            playerId: playerProfileDto.playerId,
+            firstName: playerProfileDto.firstName,
+            lastName: playerProfileDto.lastName,
+            phoneNumber: playerProfileDto.phoneNumber,
+            emailAddress: playerProfileDto.emailAddress,
+         }
+      });
+}
+
+const getTeamsListDto = (tournamentId: string): Promise<TeamsListDto> => {
+   return TeamsListRestApi()
+      .getMatchesList(tournamentId)
+      .then(teamsList => {
+         return teamsList
```

Comment:
I w sumie tutaj po raz trzeci to samo :) A w sumie to 6, bo w kazdje funkcji po 2 razy zbedne {} i return.